lib/manageiq/messaging/version.rb

```diff
@@ -1,5 +1,5 @@
 module ManageIQ
   module Messaging
-    VERSION = "0.1.6"
+    VERSION = "0.2.0"
```

Comment:
I think this would qualify as a breaking change requiring a new major version, if nothing else the connection options changed https://github.com/ManageIQ/manageiq-messaging/pull/54/files#diff-e8cbed576d737f7335ad6c641f3bf387L13-L22